packages/devtools_app/test/flutter/story_of_layout/flex_test.dart

```diff
@@ -262,4 +263,38 @@ void main() {
       );
     }, skip: kIsWeb || !isLinux);
   });
+
+  test('sum', () {
+    expect(sum([1.0, 2.0, 3.0]), 6.0);
+  });
+
+  group('computeRenderSizes', () {
+    test('shouldForceToOccupyMaxSize=false', () {
```

Comment:
To write tests that are easier to understand, you can give a name about what you expect to have happen.

For example,

test('scales renderSizes so the largest item has largestSize with forceToFill=false')

test('scales renderSizes so the items fit maxSize with forceToFill=true')